cookbooks/aws-parallelcluster-config/recipes/init.rb

```diff
@@ -23,8 +23,6 @@
 # Validate init system
 raise "Init package #{node['init_package']} not supported." unless systemd? || on_docker?
 
-include_recipe "aws-parallelcluster-config::cfnconfig_mixed"
```

Comment:
This must be kept, using: `include_recipe "aws-parallelcluster-environment::cfnconfig_mixed"`. We need to keep this step in the config phase.